app/main/views/marketplace.py

```diff
@@ -119,6 +116,15 @@ def copyright():
     return render_template('content/copyright.html')
 
 
+@main.route('/terms-of-use')
+def terms_of_use():
+    current_terms_date = current_app.config['DM_CURRENT_TERMS_DATE']
```

Comment:
Versions numbers were actually my initial approach.

Even with a number the workflow would be create new file, edit template and then change config variable.  We could remove the config variable by adding auto-detection either way.  I just didn't bother adding that in this PR.

The real difference is that it's definitely easier to increment a version number than set a timestamp.  However, I have a strong suspicion that if we used version numbers we'd only end up needing to create a mapping between version numbers and publication time, which would be more complicated than just using a timestamp in the first place.